Jenkinsfile

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env groovy
+pipeline {
+    agent any
+    environment { 
+        CC = 'clang'
+    }
+    stages {
+        stage('Test e2e') {
+            steps {
+                echo 'Testing e2e ...' + env.BRANCH_NAME
+                sh 'env'
+                sh 'make teste2e'
+            }
+        }
```

Comment:
why not do that after the build step?

Reply:
it needs to be deploy to  have an url to use in saucelabs

Reply:
I forgot i put this at the begining. It's just to avoid waiting for all the build process to pass.